en/sql/function/json_fn.rst

```diff
@@ -0,0 +1,156 @@
+:meta-keywords: cubrid json
```

Comment:
add more `:meta-keywords:` and add `:meta-description:` - they are relevant for SEO (search engine indexing).